interfaces/builtin/docker_support.go

```diff
@@ -211,6 +211,20 @@ ptrace (read, trace) peer=cri-containerd.apparmor.d,
 # what docker tries to use
 # see https://bugs.launchpad.net/snapd/+bug/1867216
 unix (bind) type=dgram,
+
+dbus (send)
+     bus=system
+     path=/org/freedesktop/systemd1
+     interface=org.freedesktop.systemd1.Manager
+     member="{StartTransientUnit,StopUnit,ResetFailedUnit,SetUnitProperties}"
+     peer=(name="org.freedesktop.systemd1"),
+
+dbus (receive)
+    bus=system
+    path=/org/freedesktop/systemd1
+    interface=org.freedesktop.systemd1.Manager
+    member=JobRemoved
+    peer=(label=unconfined),
```

Comment:
Why isn't this `peer=(name="org.freedesktop.systemd1"),` like in the one above? Or should we combine them as

    peer=(name="org.freedesktop.systemd1", label=unconfined),

?

Reply:
no, name will be the address of systemd on system bus associated with the connection, eg. name="1:123"